Formalisation(shacl)/Core/PiecesShape/DataService.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dash: <http://datashapes.org/dash#> .
+
+
+:DataServiceShape a sh:NodeShape ;
+  sh:targetClass dcat:DataService ;
+  sh:property [
+    sh:path dcat:endpointURL ;
+    sh:nodeKind sh:IRI ;
+    sh:minCount 1 ;
+    sh:name "Endpoint URL" ;
+    dash:editor dash:URIEditor ;
+    dash:viewer dash:URIViewer ;
+  ],
+  [
+    sh:path dct:title ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LabelViewer ;
+    sh:minCount 1 ;
+    sh:uniqueLang true ;
+  ],
+  [
+    sh:path dct:endpointDescription ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LabelViewer ;
+    sh:uniqueLang true ;
+  ],
+  [
+    sh:path dct:ServesDataset ;
+    sh:nodeKind dcat:Dataset ;
```

Comment:
I think dcat:Dataset could be used as a range. As following screenshot indicats.
![image](https://github.com/Health-RI/health-ri-metadata/assets/2759916/439c057b-6fb6-4f31-afea-d2458aec524c)
Is that answerns your concern?

Reply:
Sure, the range is `dcat:Dataset` according to the spec, but `sh:nodeKind` expects one of `sh:BlankNode`, `sh:IRI`, `sh:Literal`, `sh:BlankNodeOrIRI`, `sh:BlankNodeOrLiteral` or `sh:IRIOrLiteral` as its value.

The problem is that the FDP currently cannot deal with a `sh:class` or `sh:node` for `dcat:Dataset`, since it only aware of the metadata being posted, and not the entire graph.

Reply:
The model looks like below
![image](https://github.com/Health-RI/health-ri-metadata/assets/2759916/827364cd-7e4f-4b47-8cb1-cacdbcd80b03)

Can we use sh:nodeKind sh:IRI and let the user enter the link of the dataset?

Reply:
changed to sh:nodeKind sh:IRI